### PR TITLE
MPTCP: Add missing MP_CAPABLE Flags

### PR DESCRIPTION
**Pull request by RuiCunhaM**
Add missing flags in the MPTCP MP_CAPABLE option.

Fixes https://github.com/the-tcpdump-group/tcpdump/issues/933

### CODE REVIEW

File: print-mptcp.c [hunk 1]
```diff
@@ -66,6 +66,18 @@ struct mptcp_option {
 
 #define MPTCP_OPT_SUBTYPE(sub_etc)      (((sub_etc) >> 4) & 0xF)
 
+static const struct tok mp_capable_flags[] = {
+        { 0x80, "A" },
```
Comment (infrastation) [inline print-mptcp.c:70]: Perhaps `MP_CAPABLE_A`, since it exists now?

Comment (RuiCunhaM) [reply print-mptcp.c:70]: Just to make sure you mean 0x80 -> MP_CAPABLE_A ?

Comment (infrastation) [reply print-mptcp.c:70]: Yes.

Comment (RuiCunhaM) [reply print-mptcp.c:70]: Yhea it should be more clear.

**Comment by infrastation** [comment]
Thank you for preparing this change, it looks fine to me. Would anybody else like to make a look before merging?

**Comment by infrastation** [comment]
Rebased and merged, thank you!

**Pull request merged**